packages/esm-service-queues-app/src/queue-table/default-queue-table.component.tsx

```diff
@@ -22,40 +22,75 @@ import { queueTableStatusColumn } from './cells/queue-table-status-cell.componen
 import { queueTableWaitTimeColumn } from './cells/queue-table-wait-time-cell.component';
 import QueueTableExpandedRow from './queue-table-expanded-row.component';
 import QueueTable from './queue-table.component';
+import { useQueueEntries } from '../hooks/useQueueEntries';
+import { DataTableSkeleton } from '@carbon/react';
+import { type ConfigObject } from '../config-schema';
+import { queueTableVisitAttributeQueueNumberColumn } from './cells/queue-table-visit-attribute-queue-number-cell.component';
+import { activeVisitActionsColumn } from '../active-visits/active-visits-row-actions.component';
+import { useShowProviderQueueRoomModal } from '../add-provider-queue-room/add-provider-queue-room.resource';
 
 /*
 Component with default values / sub-components passed into the more generic QueueTable.
 This is used in the main dashboard of the queues app. (Currently behind a feature flag)
 */
-function DefaultQueueTable({ queueEntries }: { queueEntries: QueueEntry[] }) {
-  const layout = useLayoutType();
+function DefaultQueueTable() {
+  const selectedQueueUuid = useSelectedServiceUuid();
+  const currentLocationUuid = useSelectedQueueLocationUuid();
+  const { queueEntries, isLoading, error } = useQueueEntries({
+    queue: selectedQueueUuid,
+    location: currentLocationUuid,
+    isEnded: false,
+  });
+
   const { t } = useTranslation();
 
+  useEffect(() => {
+    if (error?.message) {
+      showSnackbar({
+        title: t('errorLoadingQueueEntries', 'Error loading queue entries'),
+        kind: 'error',
+        subtitle: error?.message,
+      });
+    }
+  }, [error?.message]);
+  const layout = useLayoutType();
+
   const [showOverlay, setShowOverlay] = useState(false);
   const [viewState, setViewState] = useState<{ selectedPatientUuid: string }>(null);
 
+  useShowProviderQueueRoomModal();
+
+  const config = useConfig<ConfigObject>();
+  const { visitQueueNumberAttributeUuid, concepts } = config;
+  const { defaultStatusConceptUuid, defaultTransitionStatus } = concepts ?? ({} as any);
+
```

Comment:
The below is consistent with what we discussed.  Might want to add a todo that this is here for backwards-compatibility pending further design of the configuation